tox.ini

```diff
@@ -1,43 +1,58 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
 envlist =
+    lint,
     py27,
-    py34,
+    pypy,
     py35,
     py36,
     py37,
-    pypy,
+    py38,
     pypy3,
-    docs,
-    coverage-report,
+    coverage
 
 [testenv]
 usedevelop = true
-commands =
-    coverage run -m zope.testrunner --test-path=src {posargs:-vc}
 deps =
-    .[test]
-    coverage
-setenv =
-    COVERAGE_FILE=.coverage.{envname}
+    zope.testrunner
+commands =
+    zope-testrunner --test-path=src []
+extras = test
 
-[testenv:docs]
-basepython = python2.7
+[testenv:lint]
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
 commands =
-    sphinx-build -ab html -d docs/_build/doctrees docs/ docs/_build/html
-    coverage run {envbindir}/sphinx-build -ab doctest -d docs/_build/doctrees docs/ docs/_build/doctest
+    flake8 src setup.py
+    check-manifest
```

Comment:
I've seen other PRs also run check-python-versions.  I think that'd be neat.

Reply:
These other PRs are generated by a newer version of the meta-script (still need a good name for it), I am going to update this PR here.